test/sagemaker_tests/mxnet/inference/integration/sagemaker/test_elastic_inference.py

```diff
@@ -23,7 +23,7 @@
 
 DEFAULT_HANDLER_PATH = os.path.join(RESOURCE_PATH, 'default_handlers')
 MODEL_PATH = os.path.join(DEFAULT_HANDLER_PATH, 'model.tar.gz')
-SCRIPT_PATH = os.path.join(DEFAULT_HANDLER_PATH, 'model', 'code', 'empty_module.py')
+SCRIPT_PATH = os.path.join(DEFAULT_HANDLER_PATH, 'model', 'code', 'eia_module.py')
```

Comment:
Is this compatible for EIA v1.5.1 image test?

Reply:
Added the logic to use different `entry_point` for `1.7.0` and `1.5.1`